pkg/plugins/builtin/ssh/README.md

```diff
@@ -42,15 +42,28 @@ action:
     ssh_key: '{{.config.mySSHKey}}'
 ```
 
-The output resulting from this configuration will be:
+## Requirements
+
+None by default. Ssh credentials should be retrieved from `{{.config.[sshkey]}}` rather than hardcoded on the template.
+
+## Note
+
+The plugin returns two objects, the `Payload` of the execution:
 
-```js
+```json
 {
   "pid": "1234",
   "uptime": "876123"
 }
 ```
 
-## Requirements
+The `Metadata` to fetch informations about plugin execution:
 
-None by default. Ssh credentials should be retrieved from `{{.config.[sshkey]}}` rather than hardcoded on the template.
\ No newline at end of file
+```json
+{
+  "output": "{\"foo\":\"bar\"}}",
```

Comment:
This is not a fitting output. Output = the raw ssh session as it was executed

Reply:
Still good now!